src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -37,6 +38,71 @@
 @Browser("Chrome_headless")
 public class SelenideAddonsTest
 {
+    private List<Runnable> runArrayWithSEREinMessage = new ArrayList<Runnable>()
+    {
+        {
+            add(
+                () -> {
+                    throw UIAssertionError.wrap(WebDriverRunner.driver(),
```

Comment:
You can shorten and adjust the first three message. They should contain the word "StaleElementReferenceException" only once and in different combinations, e.g.:
- at the beginning of the message "StaleElementReferenceException  add some words here"
- without spaces "WeCanStillSpotTheStaleElementReferenceExceptionEvenIfItTriesToHide"
- something with special characters "We think this was caused by an :StaleElementReferenceException"

Reply:
Done